### PR TITLE
[SDK-3434] Update issues templates

### PR DESCRIPTION
**Pull request by evansims**
### Changes

This PR updates the GitHub issue templates for the repository, and applies the new GitHub issue forms YML format to Bug Reports and Feature Requests.

### References

See internal ticket SDK-3434 for reference.

### Testing

Please describe how this can be tested by reviewers. Be specific about anything not tested and reasons why. If helpful, please include manual testing steps as well. 

- [ ] This change adds unit test coverage (or why not)
- [ ] This change has been tested on the latest version of the platform/language or why not

### Checklist

- [x] I have read the [Auth0 general contribution guidelines](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md)
- [x] I have read the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md)
- [x] All existing and new tests complete without errors

### CODE REVIEW

**Comment by evansims** [comment]
@Widcket 👋

- I targeted `fa` with this PR as this is the development branch you've been targeting. Please let me know if this should be pointing to `master` instead, and I'll create a fresh PR with these changes to target that.
- I know SDK-3434 mentioned pointing to `/jwt` for Community links; however, I assume `/ios` would be more appropriate for this particular library? Please let me know if I'm wrong, and I can change that.

File: .github/ISSUE_TEMPLATE/Bug Report.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
+name: 🐞 Report a bug
+description: Have you found a bug or issue? Create a bug report for this library.
```
Comment (Widcket) [inline .github/ISSUE_TEMPLATE/Bug Report.yml:2]:
```suggestion
description: Have you found a bug or issue? Create a bug report for this library
```

File: .github/ISSUE_TEMPLATE/Feature Request.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+name: 🧩 Feature request
+description: Suggest an idea or a feature for this library
+labels: ["feature request"]
+
+body:
+  - type: checkboxes
+    id: not-general-support
+    attributes:
+      label: This is not for general support and usage questions.
```
Comment (Widcket) [inline .github/ISSUE_TEMPLATE/Feature Request.yml:9]:
```suggestion
      label: This is not for general support and usage questions
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -1,5 +1,11 @@
 blank_issues_enabled: false
 contact_links:
-  - name: Auth0 Community
+  - name: 🤔 Help & Questions
     url: https://community.auth0.com/tags/c/sdks/5/ios
-    about: Discuss this library in the Auth0 Community forums
+    about: Ask general support or usage questions in the Auth0 Community forums
+  - name: 🚀 Usage
+    url: https://github.com/auth0/SimpleKeychain#usage
+    about: Check the Usage section of the README to get up to speed with SimpleKeychain.
```
Comment (Widcket) [inline .github/ISSUE_TEMPLATE/config.yml:8]:
```suggestion
    about: Check the Usage section of the README to get up to speed with SimpleKeychain
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 4]
```diff
@@ -1,5 +1,11 @@
...
+  - name: 🤔 Help & Questions
     url: https://community.auth0.com/tags/c/sdks/5/ios
-    about: Discuss this library in the Auth0 Community forums
+    about: Ask general support or usage questions in the Auth0 Community forums
+  - name: 🚀 Usage
+    url: https://github.com/auth0/SimpleKeychain#usage
+    about: Check the Usage section of the README to get up to speed with SimpleKeychain
+  - name: 📖 API Documentation
+    url: https://auth0.github.io/SimpleKeychain/
+    about: Check the public API documentation for in-depth overview of all the available features.
```
Comment (Widcket) [inline .github/ISSUE_TEMPLATE/config.yml:11]:
```suggestion
    about: Check the public API documentation for in-depth overview of all the available features
```

File: .github/ISSUE_TEMPLATE/Bug Report.yml [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+      placeholder: SPM, Cocoapods, Carthage, etc.
+    validations:
+      required: true
+
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      options:
+        - label: I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).
```
Comment (Widcket) [inline .github/ISSUE_TEMPLATE/Bug Report.yml:85]: Thanks for this. Could you please move it to the checklist at the top, as the last item?

File: .github/ISSUE_TEMPLATE/Feature Request.yml [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+      label: Additional context
+      description: Add any other context or screenshots about the feature request here.
+    validations:
+      required: false
+
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      options:
```
Comment (Widcket) [inline .github/ISSUE_TEMPLATE/Feature Request.yml:54]: Same as above, please move it to the checklist at the top.

**Review by Widcket — APPROVED** [review]
Thank you, great work! Just left a couple of comments about moving the Code of Conduct checkbox to the checklist at the top.
Also, please include the PR template from Auth0.swift (see SDK-3434).

**Comment by evansims** [comment]
@Widcket No problem, and thanks for your edits! I've updated with your feedback 👍

**Pull request merged**